RootSDK/Insurance/Services/IPolicyHolder.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using RootSDK.Insurance.Models;
+
+namespace RootSDK.Insurance.Services
+{
+    public interface IPolicyHolder
+    {
+        PolicyHolderResponse CreatePolicyHolder(object id, string firstName = null,
+            string lastName = null, string email = null,
+            string dateOfBirth = null, string cellphone = null);
+
+        IList<PolicyHolderResponse> ListPolicyHolders();
+
+        PolicyHolderResponse GetPolicyHolder(string policyHolderId);
```

Comment:
Would these methods not be better as an async task?

Reply:
Yes. ill update in the next iteration.